source/jormungandr/tests/line_sections_tests.py

```diff
@@ -638,21 +638,23 @@ def journeys(_from, to):
             self.is_valid_journey_response(r, q)
             return r
 
+        # If the journey contains stop points or lines impacted in a line section,
+        # the disruption is displayed and linked to them even if this
+        # line section is not used in the path
```

Comment:
I don't understanding the usecase for the traveler here... Isn't it gonna be confusing (for integrator or for traveler) to output disruptions impacting unused part of the journey (the same line but elsewhere) ?